app/src/cdogsService/cdogsService.js

```diff
@@ -51,6 +51,13 @@ class CdogsService {
   }
 
   async docGen(body) {
+
+    // After change to CDOGS V2
+    // The JSON data (replacements) shouldn't be enclosed in an array
+    if(Array.isArray(body.data)) {
+      body.data = body.data[0];
+    }
```

Comment:
While this works, we should be changing the examples to not have the array instead as we are expecting people to be using the json examples as a reference point. This only masks the problem instead.

Reply:
Actually, I've looked a bit more into this. 
The example template files dont use an array.
it's just the 'Data File Upload' textarea that adds them into an array.
I'll fix that instead

Reply:
I've removed the data from the array in the ui component instead.
The app isnt perfect but probably worth merging this soon as it's a fix of sorts